lxd/device/nic_bridged.go

```diff
@@ -69,6 +69,7 @@ func (d *nicBridged) validateConfig(instConf instance.ConfigReader) error {
 		"network",
 		"parent",
 		"mtu",
+		"txqlen",
```

Comment:
We should add support for this new setting to p2p and routed types too please.

Reply:
Done